core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -161,6 +163,11 @@ public HttpResponse doCompleteInstall() throws IOException, ServletException {
      * This filter will validate that the security token is provided
      */
     private final Filter FORCE_SETUP_WIZARD_FILTER = new Filter() {
+        private Pattern RESOURCE_PATTERN = Pattern.compile(".*[.](css|ttf|gif|woff|eot|png|js)");
+        private Pattern ALLOWED_PATHS = Pattern.compile(
+            "\\Q/login\\E"
+            + "|\\Q/loginError\\E"
+            + "|\\Q/securityRealm/" + new HudsonPrivateSecurityRealm(true, false, null).getAuthenticationGatewayUrl() + "\\E");
```

Comment:
OK, I see, your goal is to lighten the weight of that class in memory IIUC, and because the class itself is likely to stay as JVM rarely unload classes?